src/wallet/mod.rs

```diff
@@ -335,6 +335,15 @@ where
         self.database.borrow().get_utxo(&outpoint)
     }
 
+    /// Returns the `Tx` details corresponding to `txid` and optionally `raw_tx`
```

Comment:
For clarity I think it's OK to duplicate most of the docs from the `list_transactions` function:
```suggestion
        /// Return a single transaction made or received by the wallet
        ///
        /// Optionally fill the [`TransactionDetails::transaction`] field with the raw transaction if
        /// `include_raw` is `true`.
        ///
        /// Note that this methods only operate on the internal database, which first needs to be
        /// [`Wallet::sync`] manually.
```

Reply:
Thanks yes that makes sense.. Updated..